GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -107,8 +121,13 @@ private void folderGoUpbutton_Click(object sender, EventArgs e)
 
         private void _NO_TRANSLATE_Directory_TextChanged(object sender, EventArgs e)
         {
-            DirectoryInfo currentDirectory = new DirectoryInfo(_NO_TRANSLATE_Directory.Text);
-            folderGoUpbutton.Enabled = (currentDirectory.Parent != null);
+            if (_NO_TRANSLATE_Directory.Text.IsNotNullOrWhitespace())
+            {
+                DirectoryInfo currentDirectory = new DirectoryInfo(_NO_TRANSLATE_Directory.Text);
+                folderGoUpbutton.Enabled = currentDirectory.Parent != null;
+            }
+            else
+                folderGoUpbutton.Enabled = false;
```

Comment:
your previous version with `try...catch` was more correct, since `DirectoryInfo` can throw a number of different exceptions (https://msdn.microsoft.com/en-us/library/system.io.directoryinfo.directoryinfo(v=vs.110).aspx#Exceptions)

Perhaps something like this:

```
try 
{
    var currentDirectory = new DirectoryInfo(_NO_TRANSLATE_Directory.Text);
    folderGoUpbutton.Enabled = currentDirectory.Exists && currentDirectory.Parent != null;
}
catch
{
}

```